lib/models/zalo_login_result.dart

```diff
@@ -30,9 +32,11 @@ class ZaloLoginResult {
         userId: userId ?? this.userId,
       );
 
-  factory ZaloLoginResult.fromJson(Map<dynamic, dynamic> json) => ZaloLoginResult(
+  factory ZaloLoginResult.fromJson(Map<dynamic, dynamic> json) =>
+      ZaloLoginResult(
         oauthCode: json["oauthCode"] == null ? null : json["oauthCode"],
-        errorMessage: json["errorMessage"] == null ? null : json["errorMessage"],
+        errorMessage:
+            json["errorMessage"] == null ? null : json["errorMessage"],
         errorCode: json["errorCode"] == null ? null : json["errorCode"],
         userId: json["userId"] == null ? null : json["userId"].toString(),
       );
```

Comment:
**Suggestion:** Use explicit type instead of dynamic for better type safety in fromJson method.
```suggestion
  factory ZaloLoginResult.fromJson(Map<String, dynamic> json) =>
        ZaloLoginResult(
          oauthCode: json["oauthCode"] == null ? null : json["oauthCode"],
          errorMessage: json["errorMessage"] == null ? null : json["errorMessage"],
          errorCode: json["errorCode"] == null ? null : json["errorCode"],
          userId: json["userId"] == null ? null : json["userId"].toString(),
        );
```